src/js/components/award/RecipientInfo.jsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * RecipientInfo.jsx
+ * Created by Emily Gullo 01/20/2017
+ **/
+
+import React from 'react';
+
+const propTypes = {
+    recipient: React.PropTypes.object
+};
+
+export default class RecipientInfo extends React.Component {
+
+    render() {
+        const recipient = this.props.recipient;
+        let recipientDuns = "Not Available";
+        let businessTypeDesc = "Not Available";
+        if (recipient.recipient_unique_id) {
+            recipientDuns = recipient.recipient_unique_id;
+        }
+        if (recipient.recipient_unique_id) {
+            businessTypeDesc = recipient.recipient_unique_id;
+        }
+        const name = (
+            <div className="recipient-name">
+                {recipient.recipient_name}
+            </div>);
+        const address = (
+            <li className="recipient-address">
+                <div className="recipient-address title">
+                    Address
+                </div>
+                <div className="recipient-address name">
```

Comment:
This section does not handle foreign addresses, which the API returns with different keys. Consider updating the centralized Award model to handle foreign/domestic locations into a single, consolidated data model.